packages/applications-service-api/src/lib/config.js

```diff
@@ -9,6 +9,13 @@
 const path = require('path');
 
 module.exports = {
+	backOfficeIntegration: {
+		applications: {
+			getApplication: {
+				caseReferences: ['EN0110004', 'EN0310004']
```

Comment:
should the caserefs be in an env var so we can set different caserefs per environment?
e.g. `caseReferences: process.env.BACK_OFFICE_INTEGRATION_CASE_REFERENCES`

Reply:
I think that would be much better, perhaps we could use that env var throughout then as we add functionality.

Reply:
added in an env var. Value is a string (comma separated)